docs/dev/design_documents/client_exit_codes.md

```diff
@@ -0,0 +1,63 @@
+# Chef-Client Exit Codes
+
+Signal outside tools of specific Chef-Client run status. Ability to determine results of a Chef-Client run.
+
+## Motivation
+
+    As a Chef user,
```

Comment:
Is there something special about this formatting that needs to be kept?

Reply:
Being indented keeps markdown tools from messing it all up and gives the nice preformat boxes

Reply:
AH! Now I get it.